_data/beyond-sidebar.yml

```diff
@@ -407,24 +407,52 @@
               id: delete_shipping_method_in_shipping_zone
     - title: Customer API
       entries:
-        - title: Create customer
-          link: https://api-docs.beyondshop.cloud/ng-customer-customer-public-api.html#resources-customer-create
-          id: create_customer
-        - title: List customers
-          link: https://api-docs.beyondshop.cloud/ng-customer-customer-public-api.html#resources-customers-list
-          id: list_customers
-        - title: Show customer details
-          link: https://api-docs.beyondshop.cloud/ng-customer-customer-public-api.html#resources-customer-get
-          id: show_customer_details
-        - title: Update customer
-          link: https://api-docs.beyondshop.cloud/ng-customer-customer-public-api.html#resources-customer-update
-          id: update_customer
-        - title: Delete customer
-          link: https://api-docs.beyondshop.cloud/ng-customer-customer-public-api.html#resources-customer-delete
-          id: delete_customer
-        - title: List customer events
-          link: https://api-docs.beyondshop.cloud/ng-customer-customer-public-api.html#resources-customer-events-get
-          id: list_customer_events
+        - title: Customer
+          entries:
```

Comment:
@dkluever22 I wonder why there is a paragraph here but not in other places where we have this structure.